get-random-throw.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
remove this whitespace! make sure line 1 of your file always has something on it. you did the right thing convention-wise by leaving a line of whitespace at the end of this file. i do not know why this is the rule, but it is. 🙃